wiki/en/Tips-Tricks-More.md

```diff
@@ -98,9 +98,9 @@ Make sure you connect your MIDI device's output port to the Jamulus MIDI in port
 
 ## For Server admins
 
-### Converting a public Server to a private one on the fly
+### Converting a Registered Server to an Unregisterd one on the fly
 
-You can run a public Server long enough for your band to connect, then go private by simply unchecking the 'Make my Server Public' box in the Server GUI. Your band mates will still be connected to the Server until they disconnect. (Thanks to [David Savinkoff](https://github.com/DavidSavinkoff) for this tip!)
+You can run a Registered Server long enough for your band to connect, then go "private" by restarting your server without the `--directoryserver` option (or by setting the Directory to "none" in the Server GUI). Your band mates will still be connected to the Server until they disconnect. (Thanks to [David Savinkoff](https://github.com/DavidSavinkoff) for this tip!)
```

Comment:
I'd add `(i.e. unregistered)` after `"private"`, to explain the heading text.

Reply:
> by restarting your server without the `--directoryserver` option

You'd need to be fast to do that... I'd rather focus on the GUI and maybe JSON-RPC command (IDK if we have one which allows switching during runtime).

Reply:
I understand the clients will stay around for quite a while, minutes in some cases?

Reply:
I don't know the connection timeout, but it's probably around 30s? It's still not clean.

Reply:
Feels a bit odd not to cover the CLUI though, since I assume the vast majority will be using that.

Reply:
I agree, but using this trick is kind of odd on a CLI server. At least I wouldn't want to systemctl stop jamulus, systemctl edit jamulus, systemctl daemon-reload, systemctl start jamulus (and in the meantime half of the clients disconnected)... via CLI.

Reply:
OK - and I can't imagine anyone actually doing this switch for real rally, so I'll cut it out and CLUI reference and those users will probably be savvy enough to work it out if they wanted to.